src/LoadingOverlay.tsx

```diff
@@ -0,0 +1,40 @@
+import { EmptyState, EmptyStateIcon, Title } from '@patternfly/react-core';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Spinner/spinner';
+import { global_BackgroundColor_300 } from '@patternfly/react-tokens';
+import React, { CSSProperties } from 'react';
+
+export class LoadingOverlay extends React.Component {
+
+  spinner = () => (
+    <span className="pf-c-spinner loading-spinner" role="progressbar" aria-valuetext="Loading...">
```

Comment:
Is there a reason we can't use the PF4 react spinner?
https://www.patternfly.org/v4/documentation/react/experimental/spinner

Reply:
I didn't want to use at it mentions not intended for production use and missing typings as well.

Reply:
Ok, fair enough.